### PR TITLE
fix: Checkbox reskinning

### LINKED ISSUE #10975: [Enhancement] : Checkbox widget reskinning
https://www.notion.so/appsmith/Checkbox-b8a235faf3b74fa2aff59f68cafc0c05

### PR DESCRIPTION
**Pull request by jsartisan**
Checkbox reskinning

Fixes #10975

## Type of change
- Style updates

## How Has This Been Tested?
- Manually tested

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/client/src/components/wds/Checkbox/index.tsx [hunk 1]
```diff
@@ -1,114 +1,115 @@
...
+    // checked icon
+    &.bp3-control.bp3-checkbox input:checked ~ .bp3-control-indicator::before {
+      background-image: url(/widgets/check-icon.svg);
+    }
   `}
 `;
 
 Checkbox.defaultProps = {
-  backgroundColor: "#553DE9",
+  accentColor: "#553DE9",
```
Comment (keyurparalkar) [inline app/client/src/components/wds/Checkbox/index.tsx:112]: @jsartisan is this value needs to hard coded or does it rely on the global theme's primary color? 
If it relies on the global theme value then can we use css variables in this case?

**Review by keyurparalkar — CHANGES_REQUESTED** [review]

File: app/client/src/widgets/CheckboxGroupWidget/component/index.tsx [hunk 2]
```diff
@@ -227,12 +231,20 @@ function CheckboxGroupComponent(props: CheckboxGroupComponentProps) {
...
+                <CheckboxLabel
+                  className="t--checkbox-widget-label"
+                  disabled={isDisabled}
+                  labelPosition={LabelPosition.Left}
+                >
+                  {option.label}
+                </CheckboxLabel>
+              }
               onChange={onChange(option.value)}
-              rowSpace={rowSpace}
```
[the remark this answers is not in the document]
Comment (jsartisan) [reply app/client/src/widgets/CheckboxGroupWidget/component/index.tsx:235]: The reason is we don't need this property. Previously It was adding height equal to the space for the chbkox. But just setting height equal to the height available is the way to go here.

File: app/client/src/components/wds/Checkbox/index.tsx (hunk 1 above)
Comment (jsartisan) [reply app/client/src/components/wds/Checkbox/index.tsx:133]: Removed. I actually added these just for testing.

Comment (keyurparalkar) [reply app/client/src/components/wds/Checkbox/index.tsx]: @jsartisan it seems this is not removed yet. Can you remove it?

**Review by keyurparalkar — CHANGES_REQUESTED** [review]

**Review by keyurparalkar — APPROVED** [review]
LGTM!

**Pull request closed without merging**